teamcity: failed tests on master: testrace/TestConcurrentBatch

The following tests appear to have failed:

[#304188](https://teamcity.cockroachdb.com/viewLog.html?buildId=304188):

```
--- FAIL: testrace/TestConcurrentBatch (32.020s)
rocksdb_test.go:442: write took 10.8s
------- Stdout: -------
I170724 16:22:58.565123 267 storage/engine/rocksdb.go:402  opening rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestConcurrentBatch452262899"
I170724 16:23:30.572155 267 storage/engine/rocksdb.go:525  closing rocksdb instance at "/go/src/github.com/cockroachdb/cockroach/artifacts/TestConcurrentBatch452262899\n
```
Please assign, take a look and update the issue accordingly.

--- a/pkg/storage/engine/rocksdb_test.go
+++ b/pkg/storage/engine/rocksdb_test.go
@@ -28,6 +28,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/keys"
 	"github.com/cockroachdb/cockroach/pkg/roachpb"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
+	"github.com/cockroachdb/cockroach/pkg/util"
 	"github.com/cockroachdb/cockroach/pkg/util/encoding"
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
@@ -367,7 +368,7 @@ func TestReadAmplification(t *testing.T) {
 func TestConcurrentBatch(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 
-	if testutils.NightlyStress() {
+	if testutils.NightlyStress() || util.RaceEnabled {
 		t.Skip()
 	}
 

